providers.d/apt-mirror/provider_wrapper.sh

```diff
@@ -1,13 +1,37 @@
 bashelliteProviderWrapper() {
 
-  if [[ ! ${_r_dryrun} ]]; then
-    apt-mirror ${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf;
+  local basepath_parameter="$(grep -oP "(?<=(^set base_path[[:space:]]*))[[:alnum:]/]*" ${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf)"
+  if [[  "${basepath_parameter}" != "${_r_mirror_tld}/${_n_repo_name}" ]]; then
+    utilMsg WARN "$(utilTime)" "The \"base_path\" parameter (${basepath_parameter}) in provider.conf does not match mirror location (${_r_mirror_tld}/${_n_repo_name})..."
   fi
 
-  if [[ "${?}" != "0" ]]; then
-    utilMsg WARN "$(utilTime)" "apt-mirror either failed or completed with errors for repo (${_n_repo_name})."
+  local config_file="${_r_metadata_tld}/repos.conf.d/${_n_repo_name}/provider.conf"
+  local tmp_config_file="${HOME}/.bashellite/tmp_${_n_repo_name}_provider.conf"
+
+  cat /dev/null > ${tmp_config_file}
+
+  IFS=$'\n'
+  for line in $(cat ${config_file}); do
+    if [[ ${line} =~ ^set[[:blank:]]base_path[[:blank:]]+[[:alnum:]/]+ ]]; then
```

Comment:
Double quotes are your friend.

Reply:
We talked about not using double quotes because it doesn't work and you aren't supposed to after bash v3.2.

Reply:
As discussed, for the regex, it might not work, but consider quoting the other ${line} arg.